webviz_subsurface/plugins/_parameter_analysis/controllers/parameter_response_controller.py

```diff
@@ -590,3 +607,22 @@ def color_corr_bars(
         },
     }
     return figure
+
+
+def empty_figure() -> go.Figure:
+    return go.Figure(
+        layout=dict(
```

Comment:
Use brackets instead of `dict`. There is some performance penalty using this, and also we should try to keep the code more consistent. I.e. use brackets everywhere.